demo/api/src/app.module.ts

```diff
@@ -125,7 +125,13 @@ export class AppModule {
                 ProductsModule,
                 MenusModule,
                 DependenciesModule,
-                BrevoModule,
+                BrevoModule.register({
+                    api: {
+                        brevo: {
```

Comment:
These two levels in the config seem unnecessary to me. Why did you add them?

Reply:
I thought I could use a global config for both admin and api in the beginning that's why I nested it like this. But since admin will be a seperate package and the config cannot be used for both this is not needed. I removed the unnecessary api nesting in f3fbde7e83b8c5d0000bc1c014d19e7db94e308d